python-client/giskard/models/base/__init__.py

```diff
@@ -358,12 +396,25 @@ def download(cls, client: GiskardClient, project_key, model_id):
                     loader_class=file_meta["loader_class"],
                 )
 
+                try:
```

Comment:
This code is present twice (both in `if` and in `else`), maybe just pull it ouside of the `if else` blocks?